### DISCUSSION TITLE
Non-Sandboxed Extensions Packagable?

### DISCUSSION THREAD

**Discussion by kindpump** [TurboWarp/packager#548] [Q&A]
I'm working on a school project and have spent the last few weeks trying to figure out how to implement scratch extensions made in vm/ scratch shells into my turbowarp project in order to package it, eventually as an HTML.

Specifically, I'm looking to implement the stretch3 "HandPose2ByScratch" extension, sourced here:
https://github.com/champierre/handpose2scratch

Any help into solving this issue would be greatly appreciated! Headache inducing for sure

**Comment by kindpump** [comment]
Also I saw that Adacraft seems to be some forked version of Turbowarp, and it has the extensions built into it. Just really hoping that this means a packaged version of a project with these un-sandboxed extensions is possible.

**Reply by GarboMuffin** [reply]
Your best bet will be to ask the maintainers of those mods to make their own fork of the packager. You can tell them that making the packager easy to fork was a priority and that the README explains how to do it.

**Reply by kindpump** [reply]
If they don't respond to me or don't want to make a fork themselves would it be hard for me to do it myself using their opensource code?

**Reply by GarboMuffin** [reply]
if you are a developer, it shouldn't be hard, otherwise I am not familiar with how their mod so you will be largely on your own

the most useful resource will be https://github.com/TurboWarp/packager#tips-for-forks

(yes I know this is incredibly late, sorry about that)
